scripts/promote-cherry-picks.ts

```diff
@@ -45,23 +48,26 @@ function getChannels(ampVersion: string, currentVersions: Versions): string[] {
 
 void runPromoteJob(jobName, async () => {
   await createVersionsUpdatePullRequest((currentVersions) => {
-    const oldAmpVersion = getOldAmpVersion(ampVersion, currentVersions);
-    const channels = getChannels(oldAmpVersion, currentVersions);
+    const currentAmpVersion = getAmpVersionToCherrypick(
+      ampVersion,
+      currentVersions
+    );
+    const channels = getChannels(currentAmpVersion, currentVersions);
     const versionsChanges: {[channel: string]: string} = {};
     for (const channel of channels) {
       versionsChanges[channel] = `${Prefixes[channel]}${ampVersion}`;
     }
+    const ampVersionWithoutCherryPicksCounter = ampVersion.slice(0, 10);
+    const currentCherryPicksCount = currentAmpVersion.slice(-3);
+    const cherryPicksCount = ampVersion.slice(-3);
```

Comment:
Super-duper nit (more than free to ignore): move these outside of `createVersionsUpdatePullRequest`, for aesthetics reason only